programs/create-genesis/golos_objects.hpp

```diff
@@ -459,20 +422,23 @@ FC_REFLECT(cyberway::golos::comment_vote_object,
     (id)(voter)(comment)(orig_rshares)(rshares)(vote_percent)(auction_time)(last_update)(num_changes)
     (delegator_vote_interest_rates))
 
-
 FC_REFLECT(cyberway::golos::limit_order_object, (id)(created)(expiration)(seller)(orderid)(for_sale)(sell_price))
 FC_REFLECT(cyberway::golos::convert_request_object, (id)(owner)(requestid)(amount)(conversion_date))
-FC_REFLECT(cyberway::golos::liquidity_reward_balance_object, (id)(owner)(steem_volume)(sbd_volume)(weight)(last_update))
 FC_REFLECT(cyberway::golos::withdraw_vesting_route_object, (id)(from_account)(to_account)(percent)(auto_vest))
 FC_REFLECT(cyberway::golos::escrow_object,
     (id)(escrow_id)(from)(to)(agent)(ratification_deadline)(escrow_expiration)
     (sbd_balance)(steem_balance)(pending_fee)(to_approved)(agent_approved)(disputed))
 FC_REFLECT(cyberway::golos::savings_withdraw_object, (id)(from)(to)(memo)(request_id)(amount)(complete))
-FC_REFLECT(cyberway::golos::decline_voting_rights_request_object, (id)(account)(effective_date))
 
 FC_REFLECT(cyberway::golos::transaction_object,       BOOST_PP_SEQ_NIL)
+FC_REFLECT(cyberway::golos::block_summary_object,     BOOST_PP_SEQ_NIL)
```

Comment:
Just small comment:
empty boost preprocessor sequences can be replaced with just empty place:
```FC_REFLECT(cyberway::golos::transaction_object, )```

Reply:
this line confusing and doesn't seems right. but can be good for obfuscation : )

Reply:
@zxcat for minification : )